robot_ws/src/person_detection_robot/config/kvs_config.yaml

```diff
@@ -1,23 +1,44 @@
-# Kinesis Video Stream name. If a stream by the given name doesn't exist, it will be created.
-stream_name: "RoboMakerSampleAppPersonDetectionStream"
+kinesis_video_streamer:
+  ros__parameters:
+    aws_client_configuration:
+      # Specifies where you want the client to communicate. Examples include us-east-1 or us-west-2.
+      region: "us-east-1"
 
-# Topic name to subscribe and receive the image/video data from.
-subscription_topic: /video/encoded
+      # Values that determine the length of time, in milliseconds, to wait before timing out a request. 
+      connect_timeout_ms: 9000
+      request_timeout_ms: 9000
+      # The maximum number of allowed connections to a single server for your HTTP communications. The default value is 25.
+      # You can set this value as high as you can support the bandwidth. We recommend a value around 25.
+      #maxConnections: 25
 
-# Topic type
-  # 1: KinesisVideoFrame transport, enabling h264 streaming
-  # 2: sensor_msgs::Image transport.
-  # 3: Like 1 but with AWS Rekognition support built-in. Results will be read from 'rekognition_data_stream' and published onto 'rekognition_topic_name'.
-topic_type: 3
+      # The number of retries for connecting to AWS Services
+      max_retries: 10
 
-# Frame rate should match the encoder's (and the camera's) frame rate.
-frame_rate: 30
+    kinesis_video:
+      # This will be replaced upon the first launch with the actual path. If the path is too long, the file will be copied into /tmp/ instead.
+      log4cplus_config: "/tmp/kvs_log_configuration"
```

Comment:
When launching the robot_ws, I see errors like
```
[kinesis_video_streamer-xx] log4cplus:ERROR could not open file /tmp/kvs_log_configuration
```
I don't understand the comment here. The `kvs_config.yaml` file will be modified to replace `log4cplus_config` with another setting? And where is the code that copies `kvs_log_configuration` (presumably from `os.path.join(get_package_share_directory('kinesis_video_streamer'), 'config', 'kvs_log_configuration')`) to `/tmp/`?

Reply:
Yes, you can see that In kinesis_video_streamer package [kinesis_video_streamer.launch.py#L43](https://github.com/aws-robotics/kinesisvideo-ros2/blob/8367ffffdef41a62bc67846d217bd9658838208e/kinesis_video_streamer/launch/kinesis_video_streamer.launch.py#L43) is already copying **kvs_log_configuration** to **/tmp/**.
So, don't need to copy kvs_log_configuration from person_detection package. If need to copy from person detection package, i can implement for the same. Please input.

Reply:
Is the value of `log4cplus_config` specified in `kvs_config.yaml` overriding the logger config path that `kinesis_video_streamer.launch.py` would choose by default?

I think the `log4cplus:ERROR could not open file /tmp/kvs_log_configuration` error is occurring because `kinesis_video_streamer.launch.py` is not copying `kvs_log_configuration` to `/tmp/` under most runtime conditions, but `kvs_config.yaml` is telling the `kinesis_video_streamer` to still look for `kvs_log_configuration` under `/tmp/`.

The solution should probably be to leave the `log4cplus_config` setting specified in `kvs_config.yaml` commented out.

Reply:
Yes, after commenting out log4cplus_config in kvs_config.yaml, it is taking log4cplus_config from /opt/ros/dashing/share/kinesis_video_streamer/config/kvs_log_configuration that confirmed from /tmp//launch_params_ and it is showing

_/**:
  ros__parameters:
    log4cplus_config: /opt/ros/dashing/share/kinesis_video_streamer/config/kvs_log_configuration_

Thanks. I have done related changes in kvs_config.yaml and kinesis.launch.py and committed the same.